Update node version 18 to node 20

amitamrutiya opened the pull request:
**Notes for Reviewers**

- This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits.
4. Include before and after screenshots/terminal output.

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

l5io commented:
🚀 Preview for commit 9afb13d57fcc6bda8e6268017fe05967bbd9433a at: https://67aed9aeb0fb68fa9da4ba7c--meshery-docs-preview.netlify.app

amitamrutiya commented:
Deprecation warning present in node 18 and node 20

> cd ui; npm i; cd ..
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated lodash.get@4.4.2: This package is deprecated. Use the optional chaining (?.) operator instead.
npm warn deprecated rimraf@2.7.1: Rimraf versions prior to v4 are no longer supported
npm warn deprecated lodash.isequal@4.5.0: This package is deprecated. Use require('node:util').isDeepStrictEqual instead.
npm warn deprecated @humanwhocodes/config-array@0.13.0: Use @eslint/config-array instead
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead
npm warn deprecated glob@7.1.7: Glob versions prior to v9 are no longer supported
npm warn deprecated react-dnd-scrollzone-patch-react-17@1.0.2: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated react-sortable-tree-patch-react-17@2.9.0: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated eslint@8.57.1: This version is no longer supported. Please see https://eslint.org/version-support for other options.
npm warn deprecated @mui/base@5.0.0-beta.40-0: This package has been replaced by @base-ui-components/react

> meshery-ui@1.0.0 postinstall
> patch-package

patch-package 8.0.0
Applying patches...
No patch files found

> meshery-ui@1.0.0 prepare
> cd .. && husky install ui/.husky

husky - install command is DEPRECATED

added 1226 packages, and audited 1227 packages in 2m

353 packages are looking for funding
  run `npm fund` for details

5 vulnerabilities (1 moderate, 4 high)

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
cd provider-ui; npm i; cd ..
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated @humanwhocodes/config-array@0.13.0: Use @eslint/config-array instead
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead
npm warn deprecated eslint@8.57.1: This version is no longer supported. Please see https://eslint.org/version-support for other options.

added 482 packages, and audited 483 packages in 42s

143 packages are looking for funding
  run `npm fund` for details

1 high severity vulnerability

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.

l5io commented:
🚀 Preview for commit cf55ea7ebab9f3211697f5910096818f6ef99d1e at: https://67aedc8c699cb3bb6bf9b546--meshery-docs-preview.netlify.app

l5io commented:
🚀 Preview for commit 7ed12c26b9dde85adc2f9a7f3978d48fbee550c6 at: https://67aee2f3807285d659bea578--meshery-docs-preview.netlify.app

l5io commented:
🚀 Preview for commit 2424c839c72d0ab474fb0339bc57a5b0d914a63c at: https://67aeefac4a0798e973043710--meshery-docs-preview.netlify.app

l5io commented:
🚀 Preview for commit 29bc619a2479d10209708c49d2e2aa087c4957b7 at: https://67aef0f8e9d5092b37accb87--meshery-docs-preview.netlify.app

l5io commented:
🚀 Preview for commit 6e099568bad8a835c5c5acb90db0420faa15428e at: https://67aef3d0e14ac90d4fa929b8--meshery-docs-preview.netlify.app

l5io commented:
🚀 Preview for commit 17961dbd0afead5dd5c06ef35e7e5ed8199380c7 at: https://67aef5e3aa963bed42986cf4--meshery-docs-preview.netlify.app

l5io commented:
🚀 Preview for commit e8dcf43d756196e7b7ff5261d50718ff65df566f at: https://67af4939b969f11ad1d997cc--meshery-docs-preview.netlify.app

l5io commented:
🚀 Preview for commit f4555caa84e432301b28a91c9d7f6e6e98d8de18 at: https://67af5bfd75e1225667b02db5--meshery-docs-preview.netlify.app

l5io commented:
🚀 Preview for commit 92176eb896a82cc8d52eee4211055ffa74b54287 at: https://67af7aac1b37a7247c7cb5b7--meshery-docs-preview.netlify.app

leecalcote reviewed (approved):
lgtm
